cmd/ttn-lw-stack/commands/config.go

```diff
@@ -43,7 +44,8 @@ type Config struct {
 	JS               joinserver.Config        `name:"js"`
 	Console          console.Config           `name:"console"`
 	GCS              struct {
-		BasicStation cups.ServerConfig `name:"basic-station"`
+		BasicStation     cups.ServerConfig `name:"basic-station"`
```

Comment:
Maybe we should also alias this `cups` to `bs_cups` to avoid confusion

Reply:
Renamed in #435